travvy/src/components/AttractionsList.vue

```diff
@@ -6,16 +6,16 @@
       <!-- include the toronto heading, an image, and some info about Toronto -->
       <br><br><br>
       <!-- Generates location heading based on first recommended attraction -->
-      <h1>{{$store.state.city}}</h1>
+      <h1>{{$store.state.city}}, {{$store.state.recommendedAttractions[0].country}}</h1>
       <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
       <br>
       <h3>Description</h3>
-      <p>Toronto is the largest city in Canada, is the capital of Ontario, and is home to attractions such as the CN tower.</p>
+      <p>{{$store.state.recommendedAttractions[0].city}} is the largest city in {{$store.state.recommendedAttractions[0].country}}, and is home to attractions such as the {{$store.state.recommendedAttractions[0].attraction_name}}, {{$store.state.recommendedAttractions[1].attraction_name}}, and {{$store.state.recommendedAttractions[2].attraction_name}}.</p>
```

Comment:
We may still want to change this at some point. Not all of our cities are the largest for their countries

Reply:
agree, maybe we can have a variable for the entire sentence? aka description of each city